src/cljs/math.cljc

```diff
@@ -802,7 +802,7 @@
         f (js/Float64Array. a)
         i (js/Uint32Array. a)]
     (cond
-      (> start direction) (if-not (= start 0.0)
+      (> start direction) (if-not (zero? start)
```

Comment:
Now that I know to look for this, I note that changing `=` to `==` would have done the same thing.